src/annotation-engine/stories.tsx

```diff
@@ -251,7 +253,17 @@ const useEngineStateMachine = (
             event.event.preventDefault();
         }
     }
-
+    const keyDownEventWhileCreation = (event:KeyDownEvent , operations:Operations)=>{
+        let lastPoint = 0
+        switch(event.event.key){
+            case "Backspace":
+                lastPoint = operations.deleteLastElement()
+                styleOps.setStyleExclusivelyToPointId(hiddenStyle, lastPoint.toString());
```

Comment:
à mon avis ici il ne faut pas initialiser `let lastPoint = 0` dans ta fonction appelante `keyDownEventWhileCreation.
C'est plutôt `operations.deleteLastElement()` qui devrait avoir la responsabilité de toujours fournir une valeur cohérente.

Reply:
c'est le linter qui ne voulais pas que mette l'init dans un switch case pour pouvoir retourner le point au CSS

Reply:
Egalement, que fait `deleteLastElement()` quand il n'y a plus aucun point à supprimer ? quand il n'y en a qu'un  ? quand il y en a 2 ? Avec la manière dont la fonction utilise `splice`, sans contrôle, j'ai peur qu'on ait des résultats inattendus si on a un tableau dont la longueur est inférieure ou égale à 2.

Reply:
Non j'ai utiliser pleins de fois le clique droit pour effacer ma figure 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice
Return value
An array containing the deleted elements.

If only one element is removed, an array of one element is returned.

If no elements are removed, an empty array is returned.